.github/workflows/ci-frontend.yml

```diff
@@ -1,25 +1,39 @@
-name: Run Frontend Tests
+name: CI for frontend
+
 on:
   pull_request:
     branches:
       - master
       - develop
     paths:
       - 'frontend/**'
+
 jobs:
   test-frontend:
```

Comment:
please rename to 'ci' to be consistent with backend. Splitting for BE and FE CI works fine, good job!